games-util/gamemode/gamemode-1.5.1.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
```

Comment:
This is probably not needed. Are you sure multilib does anything with this ebuild? Just by looking at the ebuild, I have doubts (can't test yet due to systemd dep)